### PR TITLE
Musicplayer

### LINKED ISSUE #1254: Python project
**Describe the project you want to add with tech stack**
Python project - music player 

**Expected behavior**
One can listen to hindi as well as English songs , the songs can be loaded from the files and which ever is selected it will start getting played.There is on,off,pause button.One can change the songs in between also.

### PR DESCRIPTION
**Pull request by SimranLahrani842**
#1254 
python project (music player)

Close #1254

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by SimranLahrani842** [comment]
@pranjay-poddar added music player and updates the readme.md ,kindly review my PR.

**Comment by SimranLahrani842** [comment]
@pranjay-poddar added music player and updated the readme.md rows and added project title to it, kindly review my PR.

**Review by pranjay-poddar — APPROVED** [review]

**Comment by pranjay-poddar** [comment]
Resolve conflicts

**Review by pranjay-poddar — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by SimranLahrani842** [comment]
> Resolve conflicts

Done sir ,add label and gssoc and kindly review it @pranjay-poddar

**Review by pranjay-poddar — APPROVED** [review]

**Review by pranjay-poddar — CHANGES_REQUESTED** [review]
Resolve conflicts

**Comment by SimranLahrani842** [comment]
> Resolve conflicts

Please merge sir @pranjay-poddar

**Review by MohitGupta121 — APPROVED** [review]

File: Python Projects/Music Player/images.jpeg
Comment (MohitGupta121) [inline Python Projects/Music Player/images.jpeg:1]: @SimranLahrani842 please not upload images, gif in your PR. Upload them in somewhere at cloud and use that image/gif links into your project to load them.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: Python Projects/Music Player/images.jpeg
Comment (SimranLahrani842) [reply Python Projects/Music Player/images.jpeg:1]: > @SimranLahrani842 please not upload images, gif in your PR. Upload them in somewhere at cloud and use that image/gif links into your project to load them.

How to do that previously and in all other projects we have been doing the same uploading in our folder itself...

Comment (MohitGupta121) [reply Python Projects/Music Player/images.jpeg:1]: You habe to use image link not directly image path it's as simple as you add image path

Comment (SimranLahrani842) [reply Python Projects/Music Player/images.jpeg:1]: > You habe to use image link not directly image path it's as simple as you add image path

next time i ll upload the link,for this PR please merge as its been almost month now!! @pranjay-poddar and @MohitGupta121

**Comment by MohitGupta121** [comment]
@SimranLahrani842 I think you can make new PR for this I closing this as @pranjay-poddar  is busy and for merging his review is needed.

**Comment by SimranLahrani842** [comment]
> @SimranLahrani842 I think you can make new PR for this I closing this as @pranjay-poddar is busy and for merging his review is needed.

Bro how can u close the pr  it was done 1.5 months ago...I did same in other repos too.. @MohitGupta121 @pranjay-poddar

**Review by pranjay-poddar — APPROVED** [review]

**Review by pranjay-poddar — APPROVED** [review]

**Comment by pranjay-poddar** [comment]
Resolve conflicts

**Comment by SimranLahrani842** [comment]
> Resolve conflicts

Done sir🥰

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@SimranLahrani842 please remove all gif/ images from the PR and upload them in somewhere in cloud and after that use that link of gif, image to load into your project.

**Pull request closed without merging**